src/internal/utils/index.ts

```diff
@@ -27,6 +27,10 @@ export function clone<T>(o: T): T {
   return Object.assign(Object.create(Object.getPrototypeOf(o)), o);
 }
 
+export function randomNumber(): number {
+  return Date.now() * Math.floor(Math.random() * 100);
```

Comment:
why you multiply with `Date.now()`? This returned value will only increase over time lowering the range of the returned random number...?
Doesn't JS have some kind of RNG seed where you could fill in the current date?

Reply:
Well, this way the random number is also unique. As there is a change that we get the same number from the `Math.floor(Math.random() * 100)` calculation. 

> Doesn't JS have some kind of RNG seed where you could fill in the current date?

No, unfortunately not 😞

Reply:
> Well, this way the random number is also unique.

nope, not guaranteed, lets assume current date returns 2 (now) first try and 4 (a little later) a second time, then if RNG returns 2 and 1 respectively, the returned values are identical... (Of course there are countless possibilities to get to identical results)

So if uniqueness if (very) important this won't work

Reply:
You are right that it will not guaranteed that the number is unique. For example, getting the `Date.now()` can be the same as we set mostly our event listeners at the same time. However, because it return the miliseconds since epoc, the change of getting the same number is so low, that we can say we get an unique number in this case. Also, you are not setting 10000 event listeners, it will mostly `0` and up to `2`...

So in this case, this way (its a common way in `ES`) cover the purpose of getting a random (99.999% unique) number.

For the sake of more uniqueness, I updated the calculation to get more uniqueness. See https://github.com/jexia/jexia-sdk-js/pull/203/commits/7c199f0e9cb8d70ca5c82d5a175186868578fb27

Reply:
ok, if the world does not end when a value is returned twice I can live with it :smiley:

Reply:
😄